Algorithmia/client.py

```diff
@@ -7,13 +7,16 @@
 from Algorithmia.datadirectory import DataDirectory, LocalDataDirectory
 from algorithmia_api_client import Configuration, DefaultApi, ApiClient
 
+from tempfile import mkstemp
+import atexit
 import json, re, requests, six, certifi
 import os
 
 class Client(object):
     'Algorithmia Common Library'
+   
 
-    ca_cert = "./ca_cert.pem"
+    handle, ca_cert = mkstemp(suffix = ".pem")
```

Comment:
tmp file is created immediately, even if we don't use it. Also doesn't get cleaned up UNLESS we hit a branch that uses it below because the atexit handler won't be registered.